src/app/components/team/Newsletter/NewsletterRssFeed.js

```diff
@@ -87,12 +87,34 @@ const NewsletterRssFeed = ({
                     onBlur={(e) => {
                       if (e.target.value === '') {
                         onUpdateUrl(null);
+                      } else {
+                        handleLoad();
                       }
                     }}
                     value={localRssFeedUrl}
                     iconLeft={<LinkIcon />}
                     error={invalid}
-                    helpContent={helpContent || (invalid && <FormattedMessage id="newsletterRssFeed.error" defaultMessage="A valid RSS URL is required to load articles." description="Error message displayed under RSS feed URL field" />)}
+                    helpContent={
+                      helpContent ||
+                      (invalid &&
+                        <span>
+                          <FormattedMessage
+                            id="newsletterRssFeed.error"
+                            defaultMessage="This URL is not a valid RSS URL."
+                            description="Error message displayed under RSS feed URL field."
+                          />
+                          {' '}
+                          <a href="https://help.checkmedia.org/en/articles/5540430-tipline-newsletters#h_03610322d5" target="_blank" rel="noopener noreferrer" className={styles['error-label']}>
+                            <FormattedMessage
+                              id="newsletterRssFeed.learnMore"
+                              defaultMessage="Learn more"
+                              description="This is the text of a link part of an error message related to tipline newsletter RSS. Example: 'This RSS URL is invalid. Learn more.'"
+                            />
+                          </a>
+                          .
```

Comment:
The period punctuation mark should be part of the Learn more string

Reply:
Done!